easy.yaml

```diff
@@ -21,8 +21,8 @@ description: Your Description Here # Used to describe your yaml. Useful if you h
 name: YourName # Your name in-game. Spaces and underscores will be replaced with dashes
 glitches_required: # Determine the logic required to complete the seed
   none: 1 # No glitches required
-  minor_glitches: 0 # Puts fake flipper and super bunny shenanigans into logic
-  overworld_glitches: 0 # Assumes the player knows how to perform overworld glitches like fake flipper, water walk, etc
+  minor_glitches: 0 # Puts fake flipper, waterwalk, super bunny shenanigans, and etc into logic
```

Comment:
To begin, thanks for providing the review and appreciate all the feedback you have provided. It was per the guidance of one of the moderators in the discord so that is the approach I took. Here is a snippet of my thoughts from the conversation which appears to be in inline with what your proposing in the suggestion. The full conversation was held in `suggestions` on August 9th 2020.

![image](https://user-images.githubusercontent.com/42257216/89852941-04133d80-db5e-11ea-8134-dc62556f815a.png)